### PR TITLE
1.6.x

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-![License](https://img.shields.io/github/license/appwrite/sdk-for-go.svg?v=1)
-![Version](https://img.shields.io/badge/api%20version-0.5.3-blue.svg?v=1)
+![License](https://img.shields.io/github/license/appwrite/sdk-for-go.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-1.6.0-blue.svg?style=flat-square)
+[![Build Status](https://img.shields.io/travis/com/appwrite/sdk-generator?style=flat-square)](https://travis-ci.com/appwrite/sdk-generator)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord&style=flat-square)](https://appwrite.io/discord)
 
-**WORK IN PROGRESS!**
+**This SDK is compatible with Appwrite server version 1.6.x. For older versions, please check [previous releases](https://github.com/appwrite/sdk-for-go/releases).**
```
Comment (ItzNotABug) [inline README.md:9]: minor thing - SDK could be backwards compatible but there are no older published releases for `1.5.x`, right?

File: README.md [hunk 2]
```diff
@@ -21,6 +20,66 @@ To install using `go get`:
...
+## Testing the SDK
+
+* clone this repo.
+* create a project and within this project a collection.
+* configure the documents in the collection to have a _key_ = __hello__.
+* Then inject these environment variables:
+
+  ```bash
+  export YOUR_ENDPOINT=https://appwrite.io/v1  
+  export YOUR_PROJECT_ID=6…8
```
Comment (ItzNotABug) [inline README.md:32]: We moved to using the placeholder - `<YOUR_PROJECT_ID>`, right?

File: README.md [hunk 3]
```diff
@@ -21,6 +20,66 @@ To install using `go get`:
...
+
+* clone this repo.
+* create a project and within this project a collection.
+* configure the documents in the collection to have a _key_ = __hello__.
+* Then inject these environment variables:
+
+  ```bash
+  export YOUR_ENDPOINT=https://appwrite.io/v1  
+  export YOUR_PROJECT_ID=6…8  
+  export YOUR_KEY="7055781…cd95"
```
Comment (ItzNotABug) [inline README.md:33]:
```suggestion
  export YOUR_KEY="<YOUR_KEY>"  
```

File: README.md [hunk 4]
```diff
@@ -21,6 +20,66 @@ To install using `go get`:
...
 
+## Testing the SDK
+
+* clone this repo.
+* create a project and within this project a collection.
+* configure the documents in the collection to have a _key_ = __hello__.
+* Then inject these environment variables:
+
+  ```bash
+  export YOUR_ENDPOINT=https://appwrite.io/v1
```
Comment (ItzNotABug) [inline README.md:31]:
```suggestion
  export YOUR_ENDPOINT=https://cloud.appwrite.io/v1  
```

**Review by Meldiron — APPROVED** [review]

**Pull request merged**